scripts/al2/install-kernel5dot10.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -ex
+
+if [[ $AMI_TYPE == "al2kernel5dot10" || $AMI_TYPE == "al2kernel5dot10arm" ]]; then
+    sudo amazon-linux-extras install -y kernel-5.10
```

Comment:
does this need a reboot?

Reply:
Based on manual testing steps from the pull request description, reboot does not appear to be necessary here.

> Launch EC2 instance using private AMIs from above step and run uname -r to confirm Linux kernel version is 5.10.
> 
> Partial outputs (us-west-2):
>`5.10.149-133.644.amzn2.x86_64`
>`5.10.149-133.644.amzn2.aarch64`

Reply:
Discussed with @singholt offline and we agreed on the following enhancements:

- Remove kernel 4.14 rpm package as it is still visible when running `rpm -qa | grep kernel` and all references to kernel 4.14 should be removed
- Add comment in code to disclose info from this github comment: https://github.com/aws/amazon-ecs-ami/pull/97#discussion_r1029870042